test/serializer/optimized-functions/ObjectAssign7.js

```diff
@@ -0,0 +1,12 @@
+// does contain:return 25
```

Comment:
`does not contain: Object.assign` might be a better test since we want to ensure that we don't have to perform the operation. Relies less on the actual serialization form.